contracts/contracts/RequestManager.sol

```diff
@@ -764,42 +821,64 @@ contract RequestManager is Ownable, LpWhitelist, RestrictedCalls, Pausable {
     function updateToken(
         address tokenAddress,
         uint256 transferLimit,
-        uint96 minLpFee,
-        uint32 lpFeePPM,
-        uint32 protocolFeePPM
+        uint256 ethInToken
     ) external onlyOwner {
-        require(lpFeePPM <= 999_999, "Maximum PPM of 999999 exceeded");
-        require(protocolFeePPM <= 999_999, "Maximum PPM of 999999 exceeded");
-
         Token storage token = tokens[tokenAddress];
         token.transferLimit = transferLimit;
-        token.minLpFee = minLpFee;
-        token.lpFeePPM = lpFeePPM;
-        token.protocolFeePPM = protocolFeePPM;
-
-        emit TokenUpdated(
-            tokenAddress,
-            transferLimit,
-            minLpFee,
-            lpFeePPM,
-            protocolFeePPM
-        );
+        token.ethInToken = ethInToken;
+
+        emit TokenUpdated(tokenAddress, transferLimit, ethInToken);
     }
 
-    /// Set the finality period for the given target chain.
+    /// Update chain information for a given chain ID.
     ///
     /// .. note:: This function can only be called by the contract owner.
     ///
-    /// @param targetChainId The target chain ID.
-    /// @param finalityPeriod Finality period in seconds.
-    function setFinalityPeriod(
-        uint256 targetChainId,
-        uint256 finalityPeriod
+    /// @param chainId The chain ID of the chain.
+    /// @param finalityPeriod The finality period of the chain.
```

Comment:
Nitpick: I would keep the `in seconds` part that was here before, in order to make it clear that the period is in seconds.